datafusion/core/Cargo.toml

```diff
@@ -99,6 +99,9 @@ url = "2.2"
 uuid = { version = "1.0", features = ["v4"] }
 xz2 = { version = "0.1", optional = true }
 
+[target.'cfg(not(target_os = "windows"))'.dependencies]
+nix = "0.26.1"
```

Comment:
Is this dependency actually needed? 

I think it might just be used by tests (and thus could be moved to `dev-dependencies`)